Document Ephemeral Containers for 1.16 (#15994)
kubernetes/website: content/en/docs/reference/glossary/ephemeral-container.md

---
title: Ephemeral Container
id: ephemeral-container
date: 2019-08-26
full_link: /docs/concepts/workloads/pods/ephemeral-containers/
short_description: >

>   A container type that a user requests to run temporarily in a pod.

Reviewer: ```suggestion
  A type of container type that you can temporarily run inside a Pod
```

aka:
tags:
- fundamental
---

>  A container type that a user requests to run temporarily in a pod.

Reviewer: ```suggestion
A type of container that you can temporarily run inside a {{< glossary_tooltip term_id="pod" >}}.
```

>  A container type that a user requests to run temporarily in a pod.

Reviewer 2: @verb, is it more important to state that the ephemeral container is `temporarily run` or that the container is a debugging tool?
Is it correct to say, A type of container vs. A container type?
Could use a tooltip for container, {{< glossary_tooltip term_id="container" >}}

<!--more-->

> Ephemeral containers may be added to a running Pod at the request of a user to accomplish tasks such as troubleshooting. They have no resource or scheduling guarantees, and you should not use them as part of your application.

Reviewer: ```suggestion
If you want to investigate a Pod that's running with problems, you can add an ephemeral container to that Pod and carry out diagnostics. Ephemeral containers have no resource or scheduling guarantees, and you should not use them to run any part of the workload itself.
```
